pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/VarianceAggregationFunction.java

```diff
@@ -134,12 +169,7 @@ public void aggregateGroupByMV(int length, int[][] groupKeysArray, GroupByResult
 
   @Override
   public VarianceTuple extractAggregationResult(AggregationResultHolder aggregationResultHolder) {
-    VarianceTuple varianceTuple = aggregationResultHolder.getResult();
-    if (varianceTuple == null) {
-      return new VarianceTuple(0L, 0.0, 0.0);
-    } else {
-      return varianceTuple;
-    }
+    return aggregationResultHolder.getResult();
```

Comment:
Do we need to check if `_nullHandlingEnabled` is set? Will it break if it is not set and this returns `null`?

Reply:
We don't need to check if `_nullHandlingEnabled` is set here. This is because we want to return NULL (not `-infi`) if `_nullHandlingEnabled` is false and no matching row is found.

There was a bug in my previous commit: if `_nullHandlingEnabled` is false and no matching row is found, it would not return NULL. I fixed it by modifying other places.

Reply:
When null handling is disabled, we should not return `null` in the final result though. We should apply the same logic as in `AvgAggregationFunction`

Reply:
Done.